src/django_peerctl/rest/serializers/peerctl.py

```diff
@@ -1137,7 +1137,7 @@ def create(self):
             peer_port=peer_port,
             policy4_id=data.get("policy4") or None,
             policy6_id=data.get("policy6") or None,
-            status="ok",
+            status="configured",
```

Comment:
@jason20c i dont think we can default this to `configured` here because referee will post to this as well expecting the session to come out as `ok`

I think at some point you mentioned adding a status field to this serializer to control this, in hindsight i think that is the better solution - the default should be `ok`